### PR TITLE
DB: Clean up database data migrations (sql folder)

### PR DESCRIPTION
**Pull request by OskarSchamardin**
Closes issue #62.

Delete 'sql' folder found under 'be/db/dev-db/'.
Remove references to the 'sql' folder from docker-compose.
Update documentation with info related to generating migrations.

### CODE REVIEW

File: be/src/database/migrations/1619889510529-items.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
```
Comment (Orzelius) [inline be/src/database/migrations/1619889510529-items.ts:1]: What is this migration?

**Review by Kasparsu — DISMISSED** [review]

File: be/src/database/migrations/1619889510529-items.ts (hunk 1 above)
Comment (OskarSchamardin) [reply be/src/database/migrations/1619889510529-items.ts:1]: This seems to be a migration that creates a table named 'items'. That is the back end's migration. This is out of the scope of this PR. This PR was meant to remove the example data created by the database itself (unrelated to the back end). This PR is not modifying the back end's migrations.

Comment (Orzelius) [reply be/src/database/migrations/1619889510529-items.ts:1]: I know what this migration is. I'm asking why is it in this pr?

Comment (OskarSchamardin) [reply be/src/database/migrations/1619889510529-items.ts:1]: > I know what this migration is. I'm asking why is it in this pr?

It seems that i have accidentally commited it. I pulled some other branches for development. I will remove it from this PR history.

Comment (OskarSchamardin) [reply be/src/database/migrations/1619889510529-items.ts]: Accidentally added file: `be/src/database/migrations/1619889510529-items.ts` removed from history with force pushed commit: c0560007ad374e15eb16b32f6071dff1022371c2.

**Review by Orzelius — APPROVED** [review]
Never force-rewrite history while there are people reviewing the code. It's super confusing (for the reviewers) and makes them re-review all the changes, not just the latest commit. Other than that all-good.

**Comment by OskarSchamardin** [comment]
> Never force-rewrite history while there are people reviewing the code. It's super confusing (for the reviewers) and makes them re-review all the changes, not just the latest commit. Other than that all-good.

I agree, i should probably not force push commits. Would `git revert` be an alternative to removing files (that are not sensitive) from the history. [I read](https://stackoverflow.com/a/19032678) that `git revert` creates a new commit that reverses the changes of another commit. Would `git revert` be better for removing (non sensitive) files that had no business being in the history in the first place or should i just create a commit that removes the file?

**Comment by Orzelius** [comment]
> Would git revert be an alternative to removing files (that are not sensitive) from the history

Doesn't git revert revert the entire commit. I would just create a fixup commit which removes the file. Once the review process is finished I'd squash it and then merge to main.

**Review by Kasparsu — APPROVED** [review]

**Review by Erikyy — APPROVED** [review]

**Pull request merged**